rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
 # Keep this version in sync with rust/Dockerfile.
 channel = "1.81.0"
-components = ["rustfmt", "clippy"]
+components = ["rustfmt", "clippy", "llvm-tools-preview"]
```

Comment:
curious / learning - why is this needed?

Reply:
maturin uses it:

<img width="945" alt="Screenshot 2025-05-21 at 10 21 03" src="https://github.com/user-attachments/assets/be2ae636-e42e-4899-9fb8-a4a9e5aecfdd" />

https://github.com/chroma-core/chroma/actions/runs/15150757058/job/42650244932?pr=4591